react/social-logo/inc/index-header.jsx

```diff
@@ -34,11 +33,8 @@ export default React.createClass( {
 	render() {
 		const icon = this.props.icon;
 
-		const iconClass = classNames(
-			this.props.className,
-			icon,
-			'social-logo'
-		);
+		const iconClass = this.props.className ? `social-logo ${ icon } ${ this.props.className }` : `social-logo ${ icon }`;
```

Comment:
Minor, but got to thinking how we might remove duplication here... do either of the following seem appealing?

```js
const className = this.props.className;
const iconClass = `social-logo ${ icon } ${ className ? className : '' }`;
```

```js
const iconClass = [ 'social-logo', icon, this.props.className ].filter( Boolean ).join( ' ' );
```